configs/default/JHEF-JHEF405PRO.config

```diff
@@ -0,0 +1,113 @@
+# Betaflight / STM32F405 (S405) 4.2.0 Nov 22 2020 / 18:38:23 (afdac08b3) MSP API: 1.43
+
+board_name JHEF405PRO
+manufacturer_id JHEF
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 A03
+resource MOTOR 4 A02
+resource MOTOR 5 B05
+resource MOTOR 6 B07
+resource MOTOR 7 C09
+resource MOTOR 8 C08
+resource PPM 1 B08
+resource LED_STRIP 1 A09
+resource SERIAL_TX 1 B06
+resource SERIAL_TX 2 D05
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 D06
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 6 C07
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C14
+resource SPI_SCK 1 A05
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 3 C12
+resource ESCSERIAL 1 B08
+resource ADC_BATT 1 C03
+resource ADC_RSSI 1 C00
+resource ADC_CURR 1 C02
+resource FLASH_CS 1 B03
+resource OSD_CS 1 B14
+resource GYRO_EXTI 1 B13
+resource GYRO_CS 1 B12
+resource USB_DETECT 1 A08
+
+# timer
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A03 AF1
+# pin A03: TIM2 CH4 (AF1)
+timer A02 AF1
+# pin A02: TIM2 CH3 (AF1)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer A09 AF1
+# pin A09: TIM1 CH2 (AF1)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
```

Comment:
This will make SPI DMA not work in the future versions (4.3+), please try `dma ADC 1 1` or something similar and check whether that will work.

Reply:
Using `dma adc 1 1` causes the UARTs to stop working. Afraid I'm not sure how to proceed here, may have to revisit SPI DMA support later as I don't have experience in that area.

Reply:
Please try with this firmware (load config first with dma adc 1 1 included)
[betaflight_4.3.0_STM32F405_3fbf3d7a7.zip](https://github.com/betaflight/unified-targets/files/7230635/betaflight_4.3.0_STM32F405_3fbf3d7a7.zip)
:

Reply:
I found the issue - board was assigning 8 motors with timers, only 4 required for this board. Once motors 5-8 are removed the dma issues are resolved.

Reply:
@howels please try dma adc 1 1 with above firmware

Reply:
> @howels please try dma adc 1 1 with above firmware

OK, but already testing on BF 4.3 here.